pkg/agent/openflow/pipeline.go

```diff
@@ -2899,20 +2899,24 @@ func (f *featureMulticast) localMulticastForwardFlows(multicastIP net.IP, groupI
 	}
 }
 
-// externalMulticastReceiverFlow generates the flow to output multicast packets to Antrea gateway, so that local Pods can
-// send multicast packets to access the external receivers. For the case that one or more local Pods have joined the target
-// multicast group, it is handled by the flows created by function "localMulticastForwardFlows" after local Pods report the
-// IGMP membership.
+// externalMulticastReceiverFlow generates the flow to output multicast packets to Antrea gateway interface(to the host interface
```

Comment:
```suggestion
// externalMulticastReceiverFlow generates the flow to output multicast packets to Antrea gateway interface (and the host interface
```

Reply:
Updated. Personally, I think in the parentheses, the phrase `to the host interface and the uplink interface when flexibleIPAM is enabled` is more reasonable to me.